src/charm.py

```diff
@@ -50,14 +51,11 @@ def __init__(self, *args: typing.Any):
         except CharmRelationDataInvalidError as exc:
             raise RuntimeError("Invalid relation data received.") from exc
 
+        self.actions_observer = actions.Observer(self, self.state)
         self.agent_observer = agent.Observer(self, self.state)
         self.cos_observer = cos.Observer(self)
         self.ingress_observer = ingress.Observer(self)
         self.framework.observe(self.on.jenkins_pebble_ready, self._on_jenkins_pebble_ready)
-        self.framework.observe(self.on.get_admin_password_action, self._on_get_admin_password)
-        self.framework.observe(
```

Comment:
Coming from [this PR](https://github.com/canonical/jenkins-k8s-operator/pull/68). I think this line is necessary so that the `_on_storage_attached` hook is run to set the proper ownership when storage is attached

Reply:
Just realized it got deleted during the merge conflict fixes, added back! Thanks!

Reply:
@Thanhphan1147 please see https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#pr-comments-and-requests-for-changes for guidance on requesting changes in PRs